examples/source/personalization/OAuth2_Login/api.js

```diff
@@ -251,18 +251,17 @@ async function callbackForConfig(request, response, providerConfig, provider) {
  * @param {Object} options
  * @return {Object}
  */
-function fetchJson(url, options = {}) {
+async function fetchJson(url, options = {}) {
   if (!url) return;
-  return fetch(url, options)
-      .then((res) => {
-        if (!res.ok) {
-          throw new Error('Could not read response.');
-        }
-        return res.json();
-      })
-      .catch((err) => {
-        throw new Error(`An error occured fetching data: ${err.message}`);
-      });
+  try {
+    const res = await fetch(url, options);
+    if (!res.ok) {
+      throw new Error('Could not read response.');
+    }
+    return res.json();
+  } catch (err) {
+    throw new Error(`An error occured fetching data: ${err.message}`);
```

Comment:
Sorry, should've noticed this earlier: this error message obfuscates  details from the original error (e.g. request URL). I think it's fine to not catch the original exception

Reply:
So, remove the `err` and `${err.message}` and just throw the custom error?

Reply:
remove whole try / catch block

Reply:
Ok, now I see. Thanks for clarifying this.